### ISSUE TITLE
How to merge the three data sets

### ISSUE THREAD

**Issue by puyiwen** [conscienceli/SeqNet#3]
Hi, how are the three data set structures merged, can you tell me?Thank you very much!!!

**Comment by conscienceli** [comment]
This depends on your dataloader. In this implementation, you can just put them in one folder (and labels in another folder), just like one single dataset. The filename doesn't matter.

**Comment by liul-ing** [comment]
Hi,Can you tell me where the mask data should be placed?I don’t really understand how the training data  structures should be ?Thank you very much
